cmd/install.go

```diff
@@ -171,9 +188,107 @@ func (i *installer) writeResolvConf() error {
 		`nameserver 8.8.8.8
 nameserver 8.8.4.4
 `)
+
+	if i.config != nil && i.config.DNSServers != nil {
+		var s strings.Builder
+		for _, ip := range i.config.DNSServers {
+			s.WriteString("nameserver " + ip + "\n")
+		}
+		content = []byte(s.String())
+	}
 	return afero.WriteFile(i.fs, "/etc/resolv.conf", content, 0644)
 }
 
+func (i *installer) writeDNSconf() error {
+	i.log.Info("write /etc/systemd/resolved.conf.d/dns.conf")
+
+	err := i.fs.Remove("/etc/systemd/resolved.conf.d/dns.conf")
+	if err != nil {
+		i.log.Info("no /etc/systemd/resolved.conf.d/dns.conf present")
+	}
+
+	var s strings.Builder
+	s.WriteString("[Resolve]\nDNS=")
+	for _, dns := range i.config.DNSServers {
+		s.WriteString(dns + " ")
+	}
+	ts := strings.TrimRight(s.String(), " ")
+	s.Reset()
+	s.WriteString(ts)
+	s.WriteString("\nLLMNR=no")
+
+	content := []byte(s.String())
+	return afero.WriteFile(i.fs, "/etc/systemd/resolved.conf.d/dns.conf", content, 0644)
+}
+
+func (i *installer) writeNTPConf() error {
+	var ntpConfigPath string
+	var content []byte
+	var s strings.Builder
+
+	switch i.config.Role {
+	case "firewall":
+		ntpConfigPath = "/etc/chrony/chrony.conf"
+		f, err := i.fs.Open(ntpConfigPath)
+		if err != nil {
+			i.log.Info("error opening ntp config file")
+		}
+		defer f.Close()
+
+		scanner := bufio.NewScanner(f)
+		for scanner.Scan() {
+			l := scanner.Text()
+			if !strings.HasPrefix(l, "pool") {
+				s.WriteString(l + "\n")
+			}
+		}
+		for _, ntp := range i.config.NTPServers {
+			s.WriteString("server " + ntp + " iburst\n")
+		}
+	case "machine":
```

Comment:
```suggestion
	case  models.V1MachineAllocationRoleMachine:
```

There are two more occurences of "firewall" and "machine", please replace them as well